sync/upstream.py

```diff
@@ -447,6 +447,7 @@ def commit_message_filter(msg):
         metadata["gecko-reviewers"] = reviewers
     msg = commitparser.replace_reviewers(msg, "")
     msg = commitparser.strip_commit_metadata(msg)
+    msg = msg.rstrip(":;,.")
```

Comment:
I think this should work on the first line of the message, rather than on the whole thing. Also rather than hardcoding a few characters we can use a regxp like `msg.replace("", re.compile("\W*$"), re.UNICODE)` (totally untested and so wrong). But perhaps there's a good reason not to do that e.g. do we want to strip trailing full stops or question marks? Maybe it should just be some combining characters like `:` or `-` or `_`.